sr_demos/scripts/sr_demos/demo.py

```diff
@@ -154,345 +172,376 @@ def run(self):
+
+        rospy.loginfo("Welcome to the Rock, Paper, Scissors game!")
+        rospy.loginfo("The hand will count down from 3 and then you will make a gesture.")
+        rospy.loginfo("Rock is a fist, Paper is a flat hand, and Scissors is a peace sign.")
+        rospy.loginfo("Ready?")
+        rospy.sleep(3)
+
+        # Count down
+        self.execute_command_check('count_down_3', 1.0)
+        rospy.loginfo("3")
+        self.execute_command_check('count_down_2', 1.0)
+        rospy.loginfo("2")
+        self.execute_command_check('count_down_1', 1.0)
+        rospy.loginfo("1")
+        self.execute_command_check('count_down_0_ff', 1.0)
+        self.execute_command_check('count_down_0_th', 1.0)
+        rospy.loginfo("Make your gesture!")
+
+        # Select a pose at random
+        poses = ['rock', 'paper', 'scissors']
+        pose = random.choice(poses)
+        self.execute_command_check(pose, 1.0)
+        rospy.loginfo("The hand made the {} gesture!".format(pose))
+        rospy.sleep(5.0)
+
+        if pose == "rock":
+            self.commander.move_to_named_target("fingers_pack_thumb_open")
+        self.commander.move_to_named_target("open")
+        rospy.loginfo("Rock, Paper, Scissors demo completed")
+
+    def grasp_demo(self):
+        '''
+            Runs a demo that imitates grasping and squeezing an object.
+        '''
+        rospy.loginfo("Grasp Demo Started")
+
+        self.commander.move_to_named_target("open")
+        rospy.loginfo("The hand will now imitate grasping and squeezing an object...")
+        self.execute_command_check('pregrasp_pos', 2.0)
+        self.execute_command_check('grasp_pos', 5.0)
+
+        # Send all joints to current position to compensate
+        # for minor offsets created in the previous loop
+        hand_pos = {joint: degrees(i) for joint, i in robot.commander.get_joints_position().items()}
+        robot.commander.move_to_joint_value_target_unsafe(hand_pos, 2.0, wait=True, angle_degrees=True)
+        rospy.sleep(2.0)
+
+        # Generate new values to squeeze object slightly
+        offset = 5
+        squeeze = hand_pos.copy()
+        for prefix in self.prefixes:
+            squeeze.update({f"{prefix}THJ5": hand_pos[f'{prefix}THJ5'] + offset,
+                            f"{prefix}THJ2": hand_pos[f'{prefix}THJ2'] + offset,
+                            f"{prefix}FFJ3": hand_pos[f'{prefix}FFJ3'] + offset,
+                            f"{prefix}FFJ1": hand_pos[f'{prefix}FFJ1'] + offset,
+                            f"{prefix}RFJ3": hand_pos[f'{prefix}RFJ3'] + offset,
+                            f"{prefix}RFJ1": hand_pos[f'{prefix}RFJ1'] + offset})
+        if robot.hand_type == 'hand_lite' or robot.hand_type == 'hand_e':
+            for prefix in self.prefixes:
+                squeeze.update({f"{prefix}MFJ3": hand_pos[f'{prefix}MFJ3'] + offset,
+                                f"{prefix}MFJ1": hand_pos[f'{prefix}MFJ1'] + offset})
+        if robot.hand_type == 'hand_e':
```

Comment:
```suggestion
        if self.hand_type == 'hand_e':
```

Reply:
fixed in 92637be0ff846ca31b76aae33b809ec2e931900b